EDU-3012: Update Failure and Call Stack details to current Web UI (#315)
temporalio/temporal-learning: docs/getting_started/go/first_program_in_go/index.md

![The next Activity](images/activity_failure.png)

> Click any ActivityTaskFailed event link to see Failure detail information and a Call Stack trace.

Reviewer: ```suggestion
Click any `ActivityTaskFailed` event link to see the Failure details and call stack.
```

NOTE: My suggested wording better matches what is in the current UI, although the screenshot used in the tutorial is outdated.

Author: Thank you. Adopted.

Author: Thank you, adopted.
